public_html/storefront/model/checkout/extension.php

```diff
@@ -59,6 +85,14 @@ public function getExtensions($type)
         }
         ksort($output, SORT_NUMERIC);
         $this->cache->push($cache_key, $output);
+        if($type == 'shipping' && ($cartVolume || $cartWeight) && !$output){
+            $error = new AError(
+                'No shipping method found for purchase! Parcel Volume: '.$cartVolume
+                    .' cubic "'.$this->config->get('config_length_class')
+                .'", Weight: '.$cartWeight.' "'.$this->config->get('config_weight_class')
+            .'". Please check shipping extensions configurations related to volume and weight limits of parcel.');
```

Comment:
Please check the shipping extension configurations related to the products size volume and weight limits